### ISSUE TITLE
[v1.17.x] `Alt-s` (Toggle SurfingKeys on current site) stopped working

### ISSUE THREAD

**Issue by clarkwang** [brookhong/Surfingkeys#2157]
Since [settings.blocklistPattern is not working (#2154)](https://github.com/brookhong/Surfingkeys/issues/2154) I tried `<Alt-s>` but it does not work either.

**Comment by gou4shi1** [comment]
Same here.

**Comment by mizagog** [comment]
In general settings are not saved.

**Comment by brookhong** [comment]
Please turn on `Developer mode` from `chrome://extensions/` then restart your browser, it should work.

**Comment by clarkwang** [comment]
Enabling `Developer mode` does not fix the issue. Actually it's already enabled before updating to v1.17.

I tried these steps:

1. `Developer mode` is initially enabled
1. Disable `Developer mode`, then restart Chrome
1. Enable `Developer mode`, then restart Chrome
2. Now try `<Alt-s>` but it does not work for me.

**Comment by i-zanis** [comment]
Experiencing same as above with dev mode on.

**Comment by anderamko** [comment]
> Please turn on `Developer mode` from `chrome://extensions/` then restart your browser, it should work, see more #2159

This works but I can't stay whole day in dev mode.

**Comment by clarkwang** [comment]
> > Please turn on `Developer mode` from `chrome://extensions/` then restart your browser, it should work, see more #2159
> 
> This works but I can't stay whole day in dev mode.

Enabling `Developer mode` does not fix the issue for me.

**Comment by hendrikmi** [comment]
Adding to this, clicking "Disable Surfingkeys" in the menu doesn't have any effect for me either. Unfortunately, I'll need to uninstall the extension for now, as I cannot use Jupyter Notebooks while it's enabled.

**Comment by lpanebr** [comment]
Enabling  `Developer mode` does not fix the issue for me.

**Comment by aspiers** [comment]
I already had Developer mode permanently enabled.  Restarting Chrome on a newer version after an upgrade of Chrome triggered this problem.  I'm guessing the restart caused SurfingKeys to be upgraded to the new MV3 version which has this bug.  I hope this can be fixed soon because SurfingKeys is totally unusable for me without `Alt-s` functioning, as I rely heavily on native shortcuts on some sites (e.g. gmail, github).

**Comment by brookhong** [comment]
This should have been fixed in 1.17.1.

**Comment by brookhong** [comment]
Hum, yes, there is still something need to be fixed, so far `Alt-s` would just disable current site on next page load(you could see how it works after page reloading), which is not expected, would fix this on next release.

**Comment by frankzvitale** [comment]
Also here to report the same issue, `Alt-s` is no longer working for me unfortunately. This also completely breaks the extension for me since I use native Jira keyboard shortcuts

**Comment by aspiers** [comment]
The temporary workaround is to use `Alt-i` to enable pass-through, but it's a pain having to do that every time you open a site which requires it.

**Comment by aspiers** [comment]
It's actually a lot worse than I thought:

- `Alt-i` is cancelled by `Escape` so if you need to work on a website which uses `Escape` as a shortcut (and this is common), then every time you need to use it, you have to press `Escape Escape Alt-i`
- Clicking on the extension icon and disabling SurfingKeys altogether doesn't work: it's shown as disabled but is still active!

BTW this is the ninth time this issue has happened :cry:  See #101, #167, #168, #169, #170, #182, #382, and #1244 for the others.  I'm not sure why it keeps happening, but as before I **strongly** recommend coding some regression tests so that it can never happen again.

**Comment by aspiers** [comment]
I've had to disable the extension altogether :cry:  There was no other way to stop it making sites like Gmail and Figma unusable.  I really hope this issue can be fixed soon.

**Comment by NoRePercussions** [comment]
You can still [build the last public version from source](https://gist.github.com/NoRePercussions/dc56a0573d945ed625c8d4a0b4aed032). It's less desirable than having it in the web stores, but it's fully functional.

**Comment by aspiers** [comment]
> You can still [build the last public version from source](https://gist.github.com/NoRePercussions/dc56a0573d945ed625c8d4a0b4aed032). It's less desirable than having it in the web stores, but it's fully functional.

Good idea, thanks - I've done that.  Unfortunately the installation steps are not fully documented; this is fixed in #1919 but it hasn't been merged yet.

**Comment by commiyou** [comment]
Based on the f152a9a831ed4c357f072a531400d62c226be974 build. You can refer to https://developer.chrome.com/docs/extensions/get-started/tutorial/scripts-on-every-tab for installation.

[sk.zip](https://github.com/user-attachments/files/17347486/sk.zip)

**Comment by AnkushMalaker** [comment]
Is this  build supposed to fix it? It doesn't on my install @commiyou 
Infact, I can't even disable or enable this using the GUI which sounds like #2171

**Comment by commiyou** [comment]
After following the documentation and installing the zip attachment, I can disable the extension using the shortcut

**Comment by brookhong** [comment]
Fixed in 1.17.3.

**Comment by nanikit** [comment]
Is this issue really fixed? I'm currently using version 1.17.3 with the following settings and developer mode enabled:

```javascript
settings.blocklistPattern = /(github\.dev|codesandbox\.io)/i;
```

I can't use VSCodeVim on github.dev, it behaves awkwardly. I'm not sure if this happens only to me, but the only workaround I've found is to disable this extension.

**Comment by clarkwang** [comment]
> Is this issue really fixed?

It works for me. I have something like this:

~~~
settings.blocklistPattern = new RegExp('^https://(calendar|docs|drive|keep).google.com');
~~~
